src/Tasks/Microsoft.NET.Build.Tasks/NETSdkError.cs

```diff
@@ -24,13 +24,22 @@ public class NETSdkError : TaskBase
         /// </summary>
         public string[] FormatArguments { get; set; }
 
+        public bool WarningOnly { get; set; }
```

Comment:
This is a generic task that is used to generate localized errors from the Sdk.  This is the first time we needed it to generate a warning instead of an error, so I added that capability.